test/PowerShellEditorServices.Test/Session/WorkspaceTests.cs

```diff
@@ -46,7 +47,112 @@ public void CanResolveWorkspaceRelativePath()
             Assert.Equal(testPathAnotherDrive, workspace.GetRelativePath(testPathAnotherDrive));
         }
 
+        public static Workspace FixturesWorkspace()
+        {
+            return new Workspace(PowerShellVersion, Logging.NullLogger) {
+                WorkspacePath = TestUtilities.NormalizePath("Fixtures/Workspace")
+            };
+        }
+
+        public static List<string> ExecuteEnumeratePSFiles(Workspace workspace)
```

Comment:
I don't really see a lot of use for this function considering it's only used in one place. It might be ideal to use default parameters instead of method overloading the save some lines in the other tests as well.

Reply:
Fixed.  Added some static default parameters.